ca/keyutils/keyutils.go

```diff
@@ -0,0 +1,80 @@
+package keyutils
```

Comment:
https://github.com/docker/swarmkit/pull/2246/files#diff-4d81ff593f40aff60e869a25f173c15dR78 specifically hardcodes EC private keys.  

Other than that, it's more just that we don't implement converting RSA keys.